### PR TITLE
wiki configurations: Update 'Interface' attributes to new standard

### LINKED ISSUE #204: Configuration schema docs will be outdated soon
OpenTabletDriver/OpenTabletDriver#3245 changes the schema and how attributes interact.

There will be no more `MacInterface` and `WinInterface`, just `Interface`.

There will be identifier specific attributes, which currently only `Interface` and `WinUsage`? supports.

### PR DESCRIPTION
**Pull request by gonX**
Fixes #204

### CODE REVIEW

File: site/_wiki/Development/Configurations.md [hunk 1]
```diff
@@ -81,6 +81,7 @@ pinpoint devices.
 | Output Initialization Report  |       `List<byte[]>`       | A list of output reports to be sent to the device to perform the device's initialization sequence.
 |        Device Strings         | `Dictionary<byte, string>` | A list of regular expressions to be matched against specific indexes of strings contained within the device's firmware. They can be retrieved via the device string reader. This is optional, however it is commonly used to improve detection precision.
 | Initialization String Indexes |          `byte[]`          | A list of indexes to be retrieved from the device as part of the device's initialization sequence. This is optional, and very infrequently used.
+|         `Interface`           |    non-negative integer    | Specifies the USB device interface to use. For example: `0`. Should be used when the identifier might be ambiguous, or if the tablet has multiple interfaces matching the used identifiers, but only has one of them usable.
```
Comment (jamesbt365) [inline site/_wiki/Development/Configurations.md:84]: This is nested within `Attributes`, its not just sitting there raw within the identifier.

Additionally (though OTD accepts it, only the tests don't) it should be encoded as a string, so `"0"`, not `0`.

File: site/_wiki/Development/Configurations.md [hunk 2]
```diff
@@ -100,10 +101,9 @@ Some example attributes include:
...
-|    `MacInterface`    | non-negative integer | *(MacOS only)* Specifies the USB device interface to use. For example: `0`
-|    `WinInterface`    |       [00..99]       | *(Windows only)* Similar to `MacInterface`. String must have exactly two digits, (e.g. `"01"`)
 |      `WinUsage`      |       [00..99]       | *(Windows only)* Specifies the HID usage collection to use. String must have exactly two digits (e.g. `"01"`)
 | `FeatureInitDelayMs` |     milliseconds     | For tablets with multiple feature initialization reports (e.g. polling rate change), wait this many milliseconds between reports. This can help if later feature initialization reports are sometimes randomly not picked up by the tablet.
+|      `Interface`     | non-negative integer | Specifies the USB device interface to use. For example: `0`. Should be used when the identifiers might be ambiguous, or if the tablet has multiple interfaces matching the used identifiers, but only has one of them usable. We suggest using the digitizer-specific `Interface` attribute instead, if possible.
```
Comment (jamesbt365) [inline site/_wiki/Development/Configurations.md:106]: Ditto

File: site/_wiki/Development/Configurations.md (hunk 1 above)
Comment (gonX) [reply site/_wiki/Development/Configurations.md:84]: Whoops, didn't notice the column header differed. And good catch on it being nested.

File: site/_wiki/Development/Configurations.md (hunk 2 above)
Comment (gonX) [reply site/_wiki/Development/Configurations.md:106]: The column header explicitly states string value, if you mean ditto in the sense that only non-negative integers are taken. I would be open for making the column headers more consistent in a separate PR.

Comment (jamesbt365) [reply site/_wiki/Development/Configurations.md:106]: Nah, turns out I just suck at reading.

File: site/_wiki/Development/Configurations.md (hunk 1 above)
Comment (gonX) [reply site/_wiki/Development/Configurations.md]: I think this is resolved with 59b8d9c

**Review by jamesbt365 — APPROVED** [review]

**Comment by gonX** [comment]
Forgot to add a screenshot for the new table:
image

**Pull request merged**